odo/backends/tests/test_postgres.py

```diff
@@ -383,17 +383,40 @@ def test_to_dataframe_strings(sql_with_strings):
     sql_with_strings, bind = sql_with_strings
 
     insert_query = sql_with_strings.insert().values([
+        # quoting is done correctly
         {'non_optional': 'ayy', 'optional': 'hello "world"'},
+
+        # None stays None
         {'non_optional': 'lmao', 'optional': None},
+
+        # nan string is just the string nan
+        {'non_optional': 'nan', 'optional': 'this ticker sucks'},
+
+        # NULL is just the string NULL
+        {'non_optional': 'NULL', 'optional': 'just checking'},
+
+        # escaping is done correctly
+        {'non_optional': r'\no\t \ \escaped \"',
+         'optional': r'\no\t \ \escaped \"'},
+
+        # empty string is not NULL
+        {'non_optional': '', 'optional': ''},
     ])
     if bind is None:
         insert_query.execute()
     else:
         bind.execute(insert_query)
 
+    expected = pd.DataFrame(
+        [['ayy', 'hello "world"'],
+         ['lmao', None],
+         ['nan', 'this ticker sucks'],
```

Comment:
We discussed in person about testing `nan` and `NULL` on both sides of the test, other than that, LGTM.
If test failures you are seeing on the dependent project require more work, let me know and can review those changes.